explore/src/main/scala/explore/programs/ProgramDetailsTile.scala

```diff
@@ -17,52 +21,94 @@ import lucuma.core.model.Program
-        FormInfo(if (thesis) "Yes" else "No", "Thesis"),
-        FormInfo(s"${details.proprietaryMonths} months", "Proprietary")
-      ),
-      <.div(
-        TimeAwardTable(details.allocations),
-        TimeAccountingTable(props.programTimes)
-      ),
-      <.div(ExploreStyles.ProgramDetailsInfoArea)(
-        SupportUsers(
-          props.programId,
-          users,
-          "Principal Support",
-          SupportUsers.SupportRole.Primary
+      val details: ProgramDetails        = props.programDetails.get
+      val thesis: Boolean                = details.allUsers.exists(_.thesis.exists(_ === true))
+      val users: View[List[ProgramUser]] = props.programDetails.zoom(ProgramDetails.allUsers)
+      val newDataNotificationView        =
+        props.programDetails
+          .zoom(ProgramDetails.shouldNotify)
+          .withOnMod(b => ProgramQueries.updateGoaShouldNotify[IO](props.programId, b).runAsync)
+
+      <.div(ExploreStyles.ProgramDetailsTile)(
+        <.div(ExploreStyles.ProgramDetailsInfoArea, ExploreStyles.ProgramDetailsLeft)(
+          FormInfo(details.reference.map(_.label).getOrElse("---"), "Reference"),
+          FormInfo(Constants.GppDateFormatter.format(props.semester.start.localDate), "Start"),
+          FormInfo(Constants.GppDateFormatter.format(props.semester.end.localDate), "End"),
+          // Thesis should be set True if any of the investigators will use the proposal as part of their thesis (3390)
+          FormInfo(if (thesis) "Yes" else "No", "Thesis"),
+          FormInfo(s"${details.proprietaryMonths} months", "Proprietary")
+        ),
+        <.div(
+          TimeAwardTable(details.allocations),
+          TimeAccountingTable(props.programTimes)
         ),
-        SupportUsers(
-          props.programId,
-          users,
-          "Additional Support",
-          SupportUsers.SupportRole.Secondary
+        <.div(ExploreStyles.ProgramDetailsInfoArea)(
+          SupportUsers(
+            props.programId,
+            users,
+            "Principal Support",
+            SupportUsers.SupportRole.Primary
+          ),
+          SupportUsers(
+            props.programId,
+            users,
+            "Additional Support",
+            SupportUsers.SupportRole.Secondary
+          ),
+
+          // The two Notifications flags are user-settable and determine whether the archive sends email notifications for new data and whether the ODB sends notifications for expired timing windows (3388, 3389)
+          <.div(
+            ExploreStyles.ProgramDetailsRight,
+            FormInfo(
+              CheckboxView(
+                id = "shouldNotify".refined,
+                value = newDataNotificationView,
+                label = "New Science Data",
+                disabled = props.userIsReadonlyCoi
+              ),
+              "Notifications"
+            )
+            // FormInfo(
```

Comment:
Commented code

Reply:
I was planning on leaving the commented code here because it provides the layout for the timing windows notifications and eavesdropping features that are in the mockups.